.github/workflows/check-media-attachments.yml

```
@@ -0,0 +1,160 @@
+                if (imgWithoutAlt && imgWithoutAlt.length > 0) {
+                  hasIssues = true;
+                  issues.push(`**${file.filename}**: Found ${imgWithoutAlt.length} image(s) without alt attributes`);
+                  console.log(`  - Found ${imgWithoutAlt.length} img tag(s) without alt attributes`);
+                }
+                
+                // Check for media files referenced
+                const mediaMatches = content.match(/(src|href)=["'][^"']*\.(jpg|jpeg|png|gif|webp|svg|mp4|avi|mov|pdf)["']/gi);
+                if (mediaMatches && mediaMatches.length > 0) {
+                  mediaReferences.push(`**${file.filename}**: References ${mediaMatches.length} media file(s)`);
+                  console.log(`  - Found ${mediaMatches.length} media reference(s)`);
+                  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)